From cfc6839362bd96bf2f42954160f16a60bf1de377 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 22 Dec 2018 19:59:02 -0800
Subject: [PATCH] misc: Do not print NULL string into logs

string format %s is getting a NULL pointer for 'expand'
parameter always since the check for chunks[i] == NULL will ensure that
its always null when the Log() API is called

Upstream-Status: Submitted [https://github.com/vmware/open-vm-tools/pull/312]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 open-vm-tools/lib/misc/util_misc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/open-vm-tools/lib/misc/util_misc.c
+++ b/open-vm-tools/lib/misc/util_misc.c
@@ -719,7 +719,7 @@ Util_ExpandString(const char *fileName)
       ASSERT(!freeChunk[i]);
       chunks[i] = expand;
       if (chunks[i] == NULL) {
-	 Log("%s: Cannot allocate memory to expand $ in \"%s\".\n",
+	 Log("%s: Cannot allocate memory to expand in \"%s\".\n",
              __FUNCTION__, fileName);
 	 goto out;
       }
